### DISCUSSION TITLE
How to create and run tauri app in windows without visual studio ?

### DISCUSSION THREAD

**Discussion by Jigar-Gadhia** [tauri-apps/tauri#8171] [Q&A]
I installed and configured minGW for windows build tools and created tauri app using yarn and tried to run but I am getting error: 

> error: failed to run custom build command for `Remindify v0.0.0 (F:\new project\Remindify v0.1\src-tauri)`
note: To improve backtraces for build dependencies, set the CARGO_PROFILE_RELEASE_BUILD_OVERRIDE_DEBUG=true environment variable to enable debug information generation.

Caused by:
  process didn't exit successfully: `F:\new project\Remindify v0.1\src-tauri\target\release\build\Remindify-9201b6277228d806\build-script-build` (exit code: 101)
  --- stdout
  cargo:rerun-if-env-changed=TAURI_CONFIG
  cargo:rerun-if-changed=tauri.conf.json
  cargo:rustc-cfg=desktop
  package.metadata does not exist

  --- stderr
  windres: Can't detect target endianness and architecture.
  thread 'main' panicked at C:\Users\jayga\.cargo\registry\src\index.crates.io-6f17d22bba15001f\embed-resource-2.4.0\src\windows_not_msvc.rs:48:25:
  windres failed to compile "F:\new project\Remindify v0.1\src-tauri\target\release\build\Remindify-8d8702221404ce8d\out\resource.rc" into "F:\new project\Remindify v0.1\src-tauri\target\release\build\Remindify-8d8702221404ce8d\out/libresource.a" with exit code: 1
  stack backtrace:
     0: rust_begin_unwind
               at /rustc/cc66ad468955717ab92600c770da8c1601a4ff33/library\std\src/panicking.rs:595:5
     1: core::panicking::panic_fmt
               at /rustc/cc66ad468955717ab92600c770da8c1601a4ff33/library\core\src/panicking.rs:67:14
     2: embed_resource::windows_not_msvc::ResourceCompiler::compile_resource
     3: embed_resource::compile_impl
     4: embed_resource::compile
     5: tauri_winres::WindowsResource::compile
     6: tauri_build::try_build
     7: tauri_build::build
     8: build_script_build::main
     9: core::ops::function::FnOnce::call_once
  note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.

The default channel for rust is stable-x86_64-pc-windows-gnu (default).

Please somebody help me to solve this issue

**Comment by FabianLars** [comment]
Tauri does not officially support the gnu toolchain on Windows so i don't have many ideas here. The main one though, is that i think you're simply missing the resource compiler. Did you install the `binutils` package in your msys2 instance (Not sure about the exact package name, something like `mingw-w64-binutils` maybe)?
